src/models/scenario.ts

```diff
@@ -52,6 +53,7 @@ export class Scenario {
 
     this.tokens = Object.assign({}, (scenario.tokens || {}), tokenOverrides)
     let rawSteps = this.applyTokens(scenario.steps, this.tokens)
+    rawSteps = this.replaceSpecialDateTokens(rawSteps)
```

Comment:
A thought!  It's possible that someone could supply a date token that does not parse to a valid date object (e.g. `{{date(this century)}}`.  Currently, this results in an obtuse error:

![Screen Shot 2020-06-09 at 2 01 10 PM](https://user-images.githubusercontent.com/3496491/84199365-e5a79d80-aa59-11ea-899c-bbe148d89a92.png)

Perhaps we try/catch and throw an `InvalidScenarioError` with a message like `Unable to parse token (${tokenText}) as date (${fromFile})`